### PR TITLE
Updates to questnet rules

### PR DESCRIPTION
**Pull request by Afauvre**
Updated based on previous winners and upcoming May competition

### CODE REVIEW

**Comment by Afauvre** [comment]
@pro-wh to add final formula

File: src/operators/the-quest-rules.md [hunk 1]
```diff
@@ -69,73 +71,60 @@ Before diving into your strategy for all items listed below, we also encourage
...
-    that point, we will announce the top 5 operators with the most blocks signed
-    as well as the special prize for most blocks signed from the nodes who
-    joined after Jan 23, 2020.
-* **Best Attacks**: Best attacks is a qualitative reward but certainly one we
+* **May Availability Challenge:** Availability is a measure of how well
+  a node operator is performing. The May competition will run from May
+  11 to June 4.
+  
+  For May, we define
+  ``Best Availability (v3) = Blocks Signed + [to update`
```
Comment (pro-wh) [inline src/operators/the-quest-rules.md:81]: (me) put formula here

**Comment by pro-wh** [comment]
I've filled in a proposal for availability v3. Have a  look.

File: src/operators/the-quest-rules.md [hunk 2]
```diff
@@ -1,36 +1,37 @@
...
+# Rules and Guidelines for The Quest
 
 ![The Quest](/operator_images/the_quest.png)
 
 Welcome to The Quest -- a multi-round staking competition (aka incentivized
-testnet) run on the Oasis Network. The Quest began on January 15 00:00 UTC.
-**New participants can join during or after the next network upgrade on
-Thursday, January 23 at 00:00 UTC (Wednesday, January 22 at 4:00 pm PST).**
+testnet) run on the Oasis Network. We are currently in the May Availability
+Challenge of The Quest. **The May competition will start on May 11 at
```
Comment (peterjgilbert) [inline src/operators/the-quest-rules.md:7]: Is this PR intended to merge after that?

File: src/operators/the-quest-rules.md [hunk 3]
```diff
@@ -185,17 +169,28 @@ turned on):
 * Best team player: up to 100,000 tokens
 * Community content: up to 100,000 tokens
 
-**Participation**:
+**Availability Participation**:
```
Comment (peterjgilbert) [inline src/operators/the-quest-rules.md:172]:
```suggestion
**Best Availability and Participation**:
```

**Pull request merged**